### PR TITLE
Small spelling corrections and semantic correction in the proof section

### PR DESCRIPTION
**Pull request by digitsu**
Corrected 'mining' to 'minting'
Corrected 'b' to 'a' in the part talking about the a _lessdot_ b
Removed excess whitespaces

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -99,12 +99,12 @@ \subsection{Genesis Transactions}\label{subsec:gentrans}
 
 This metadata instantiates variables used in the token's consensus rules. For example if a burn address is provided, the ownership transaction format should be changed to allow for the burn address to revoke, but never assert ownership.\footnote{At the time of this writing, this feature has not yet been implemented}
 
-Mining delegation is accomplished via an output whose pubkey script is of the form:
+Minting delegation is accomplished via an output whose pubkey script is of the form:
```
Comment (TheRealBluesun) [inline main.tex:102]: Good catch

File: main.tex [hunk 2]
```diff
@@ -139,17 +139,17 @@ \section{Consensus Rules}
...
+
 \section{Proofs}
 While possible, it is undesirable to trace back through the transaction graph to the genesis transaction and verify the consensus rules hold as it precludes SPV compatibility. We solve this conundrum by supplementing each primitive transaction with a proof that the revoke and mint signature inputs are spending assert and mint pubkey outputs within valid ownership or minting transactions respectively. In the final incarnation proofs will validate the entire transaction graph by verifying proofs given \emph{only} in the prior primitive transactions by using recursion.
 
 \begin{center}
     \import{res/}{basictxproofs}
 \end{center}
 
-We adopt notation from order theory where $a \lessdot b$ means that transaction $b$ has a (mint signature and revoke) input spending transaction $b$'s (mint pubkey and assert) output.  Additionally, $a < b$ means there exists a sequence $a \lessdot \dots \lessdot b$.
+We adopt notation from order theory where $a \lessdot b$ means that transaction $b$ has a (mint signature and revoke) input spending transaction $a$'s (mint pubkey and assert) output.  Additionally, $a < b$ means there exists a sequence $a \lessdot \dots \lessdot b$.
```
Comment (TheRealBluesun) [inline main.tex:152]: Another good catch -- thank you.

**Review by TheRealBluesun — APPROVED** [review]
Thanks for your contribution!  A fresh pair of eyes is always good.

**Comment by digitsu** [comment]
Looking forward to seeing the proof engine so that we can start building on
this.

**Comment by recursivesmelting** [comment]
In early stages of prototyping but you can follow/help with development [here](https://github.com/recursivesmelting/rs-proof-lib) :)

**Pull request merged**